src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -558,4 +565,40 @@ private async Task HandleOnRowFocusAsync(DataGridRowFocusEventArgs args)
             }
         }
     }
+
+    public Task OnKeyDownAsync(FluentKeyCodeEventArgs args)
+    {
+        if (args.ShiftKey == true && args.Key == KeyCode.KeyR)
+        {
+            ResetColumnWidths();
+        }
+
+        if (args.Key == KeyCode.NumpadSubtract || args.Key == KeyCode.Minus2)
+        {
+            SetColumnWidth(-10);
+        }
+        if (args.Key == KeyCode.NumpadAdd || (args.ShiftKey == true && (args.Key == KeyCode.Equal || args.Key == KeyCode.Equal2)))
+        {
+            //  Resize column up
+            SetColumnWidth(10);
+        }
+        return Task.CompletedTask;
+    }
+
+    private void SetColumnWidth(float widthChange)
+    {
+        if (_gridReference is not null)
+        {
+            _ = Module?.InvokeVoidAsync("resizeColumn", _gridReference, widthChange).AsTask();
```

Comment:
Why are you using `AsTask`. I prefer to use `async / await`, transforming the `SetColumnWidth` to an async method (suffix Async)